replacing-databases-bigquery/cloudbuild.yaml

```
@@ -0,0 +1,39 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- id: 'tf init'
+  name: 'hashicorp/terraform:1.0.0'
+  entrypoint: 'sh'
+  args: 
+  - '-c'
+  - | 
+    terraform init \
+    -backend-config="bucket=$PROJECT_ID-tf-state" \
+    -backend-config="prefix=replacing-databases-bigquery"
+  dir: terraform
+
+- id: 'tf apply'
+  name: 'fellipemedeiros/terraformgcloud:latest'
```

```suggestion
  name: 'hashicorp/terraform:1.0.0'
```
could we use hashicorp image?

---

i change the image because we need need the gcloud to run some commands (this problem only happens when we run on cloudbuild)